src/components/Footer/FooterLinksGroup.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Typography } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+interface Links {
+  title: string;
+  routeLink: string;
+}
+
+interface FooterLinksGroupProps {
+  groupTitle: string;
+  links: Array<Links>;
```

Comment:
FYI you can write links type as well: Links[], looks more simple. but you don'y need to fix it, it's just FYI